test/tools/invoker.py

```diff
@@ -11,7 +11,7 @@ class UnittestInvoker:
     # and cleans up the generated descriptor_set file.
     _CURRENT_DIR = os.getcwd()
     _PROTOS_DIR = os.path.join(_CURRENT_DIR, "test/testdata/protos/example/")
-    _PROTOC = os.path.join(_CURRENT_DIR, f'test/tools/protoc')
+    _PROTOC = os.path.join(_CURRENT_DIR, f"test/tools/protoc")
```

Comment:
There are some minor gotchas with `f`strings, but in general I completely agree.

Reply:
One thing I noticed that if we use `f""`, the linter could not fold the whole string into lines. But I'm open to replace the `string.format()` to `f""` if necessary. And this will be done in separate PR. 
But in this case, `f""` is not needed, removed it. Thanks!